api/conf/conf.go

```diff
@@ -69,8 +70,14 @@ type ErrorLog struct {
 	FilePath string `yaml:"file_path"`
 }
 
+type AccessLog struct {
+	Level    string
```

Comment:
log level in access log is no sense.

Reply:
You can change the zapcore encoder to remove the level encoding.

Reply:
fixed.